ci_scripts/configure-tde-server.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
+
 set -e
 
 # This script is used to configure a TDE server for testing purposes.
 export TDE_MODE=1
+export PERCONA_SERVER_VERSION=17.4.1
```

Comment:
Why do we need the variable for the configure command? It is only used by test commands